app/initializers/setup.js

```diff
@@ -1,20 +1,13 @@
-import { typeOf } from '@ember/utils';
 import config from 'ember-inspector/config/environment';
 import PromiseAssembler from "ember-inspector/libs/promise-assembler";
 
 export default {
   name: 'setup',
   initialize(instance) {
-    // {{EMBER_DIST}} is replaced by the build process.
-    instance.adapter = '{{EMBER_DIST}}';
+    // {{EMBER_DIST}} is replaced by the build process (basic, chrome, etc)
+    let Adapter = instance.resolveRegistration(`adapter:{{EMBER_DIST}}`);
 
     // register and inject adapter
-    let Adapter;
-    if (typeOf(instance.adapter) === 'string') {
-      Adapter = instance.resolveRegistration(`adapter:${instance.adapter}`);
-    } else {
```

Comment:
I don't think this branch is possible: on line 9 we unconditionally set it to a string, so seems impossible for it to fall through.